libkbfs/kbfs_ops.go

```diff
@@ -185,6 +185,10 @@ func (fs *KBFSOpsStandard) DeleteFavorite(ctx context.Context,
 }
 
 func (fs *KBFSOpsStandard) getOpsNoAdd(fb FolderBranch) *folderBranchOps {
+	if fb == (FolderBranch{}) {
+		panic("zero FolderBranch in getOps")
```

Comment:
I figured this was more fool-proof than trying to plumb the error up from everywhere.